.github/workflows/check-catalyst.yaml

```diff
@@ -468,6 +468,19 @@ jobs:
         run: |
           sudo apt-get -y -q install cmake ninja-build libomp-dev lcov
 
+      - name: Install awscli (OpenQasm device)
+        if: ${{ matrix.simulator == 'openqasm' }}
+        id: install-aws-cli
+        uses: unfor19/install-aws-cli-action@v1
```

Comment:
Do we really need this action? Just asking since it's a small third party action and it's good to avoid third party code if we don't need it.

Reply:
I used this action to avoid installing and configuring `awscli` in our end. I don't mind removing it though.